schemas/fingerprint-server-api.yaml

```
@@ -857,6 +857,9 @@ components:
           format: float
           minimum: 0
           maximum: 1
+        revision:
+          description: The version of the method used to calculate the Confidence score. 
```

Maybe let's explain that this is only present for select customers? 

```suggestion
          description: The version of the method used to calculate the Confidence score. This field is only present for customers who opted into alternative calculation methods.
```